README.md

```diff
@@ -1,3 +1,7 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Here is a link to my loom video [Project description video](https://www.loom.com/share/7a461598ef784fe1aa1b03ecf7f3bf13)
+
 # Here is a link to my loom video [Project description video]()
 
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] As you can see this badge is repeated two times which is not good practice but also it will not look very nice on the README. Please remove this one ☝️

Reply:
@BertrandConxy Noted. Thank you